main.cpp

```diff
@@ -18,16 +18,45 @@ using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
+private:
+    Rewriter& rewriter_;
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
+    CastCallBack(Rewriter& rewriter) : rewriter_(rewriter) { }
 
     void run(const MatchFinder::MatchResult &Result) override {
         // Your code goes here
+        const auto *Item = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `Item` is a null pointer? How to protect the program from crashing in such cases?

Reply:
If Item is a null pointer the program will crash. To avoid this, it is necessary to check Item for a non-zero value.
if (Item == nullptr) {
    return;
}